chain/jsonrpc/res/epoch_info.html

```diff
@@ -0,0 +1,86 @@
+<html>
+<head>
+    <style>
+        table {
+            width: 100%;
+            border-collapse: collapse;
+        }
+
+        table,
+        th,
+        td {
+            border: 1px solid black;
+        }
+
+        td {
+            text-align: left;
+            vertical-align: top;
+            padding: 8px;
+        }
+
+        th {
+            text-align: center;
+            vertical-align: center;
+            padding: 8px;
+            background-color: lightgrey;
+        }
+
+        tr.active {
+            background-color: #eff8bf;
+        }
+    </style>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script>
+        $(document).ready(() => {
+            $('span').text("Loading...");
+            $.ajax({
+                type: "GET",
+                url: "/debug/api/status",
+                success: data => {
+                    let epoch_info = data.detailed_debug_status.epoch_info;
+                    $('.js-epoch-id').text(epoch_info.epoch_id);
+                    $('.js-epoch-start-time').text(epoch_info.start_time);
+                    epoch_info.validators.forEach((validator, index) =>
+                        $('.js-tbody-validators').append($('<tr>')
+                            .append($('<td>').append(validator.account_id))
+                        )
+                    );
+                },
+                dataType: "json",
+                error: function (errMsg, textStatus, errorThrown) {
+                    alert("Failed: " + textStatus + " :" + errorThrown);
+                },
+                contentType: "application/json; charset=utf-8",
+            })
+
+        });
+    </script>
+</head>
+<body>
+    <h1>
+        Welcome to the Epoch Status page!
+    </h1>
+    <h2>
+        <p>
+            Current Epoch ID:
+            <span class="js-epoch-id"></span>
+        </p>
+        <p>
+            First block started at:
+            <span class="js-epoch-start-time"></span>
```

Comment:
can we also print the first block height and block hash ?

Reply:
And how many blocks have been produced already in this epoch, and the number of blocks left